.changeset/chatty-teachers-ring.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fuel-ts/abi-typegen": patch
+"@fuel-ts/abi-coder": patch
+"@fuel-ts/versions": patch
+"@fuel-ts/script": patch
+"@fuel-ts/utils": patch
+"@fuel-ts/forc": patch
```

Comment:
1. All the packages are `patch`, should they be `minor`?
2. If there are plans to merge #2143 into this release, maybe we shouldn't because this changeset is updating the forc package.

Reply:
1. Maybe, why you say?
2. No.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2180/commits/e4271bc2ab42ca9f0fbc08a62657537efedca097

Reply:
Because I didn't ask the [previous](https://github.com/FuelLabs/fuels-ts/pull/2045/files#diff-184a46055a49bfe0a742cbc5bd8b3bea2a2c99cac5fa1df5881f236a0a08b8f6) time haha. It could've gone with `patch` as well, it doesn't matter. I wrote the comment mainly because of the second point.